### DISCUSSION TITLE
Исправления в озвучке?

### DISCUSSION THREAD

**Discussion by kot9ka** [dosinabox/g2nr_unofficial_update#512] [General]
Мне нужно устанавливать исправления в озвучке? Что это такое если можно на примерах.

**Comment by dosinabox** [comment]
Описание и список изменений: https://worldofplayers.ru/threads/36819/
Для стим-версии устанавливать его не нужно, там официально включили в контент.

**Comment by kot9ka** [comment]
Тогда ясно. У меня куплена в стиме. Я думал там исправили в начале игре слово ПОИСК, ПОИСКЕ три раза подряд. Такая бредятина. Я думал может в этом патче подправили. Значит нет.
Я только закончил прохождение версии без аддона. Решил пройти Ночь Ворона. Но эта озвучка уже в самом начале игры говорит мне что наверно не стоит)) Если такое в начале то что будет дальше...

**Reply by dosinabox** [reply]
ты про что?

**Reply by kot9ka** [reply]
> ты про что?
ая яй. я один кат-сцены смотрю в играх..))
https://www.youtube.com/watch?v=O0hU1lvE3CQ&t=194s
самое начало игры.

**Reply by dosinabox** [reply]
видео никто не трогал, к исправлению озвучки это не имеет отношения
